docker-compose.yml

```diff
@@ -34,7 +34,16 @@ services:
     depends_on:
       - database
     command:
-      - start-dev --import-realm --spi-theme-static-max-age=-1 --spi-theme-cache-themes=false --spi-theme-cache-templates=false --spi-theme-welcome-theme=uid2-theme --health-enabled=true --metrics-enabled=true --features=declarative-user-profile --spi-connections-jpa-legacy-migration-strategy=update
+      - start-dev
```

Comment:
Arguments need to be passed in separately rather than a single quoted value of all arguments, per: https://www.keycloak.org/docs/latest/upgrading/index.html#kc-sh-and-shell-metacharacters